app.rb

```diff
@@ -0,0 +1,94 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './rental'
+require './book'
+
+class App
+  def initialize()
+    @all_books = []
+    @all_persons = []
+  end
+
+  def create_book(title, author)
+    book = Book.new(title, author)
+    @all_books.push(book)
+    puts 'Book created successfully'
+  end
+
+  def create_person(person_code)
+    if person_code == 1
+      create_student
+    elsif person_code == 2
+      create_teacher
+    else
+      puts 'Enter valid number'
+      call(3)
+    end
+    puts 'Person created successfully'
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp
+    permission_values = %w[n N]
+    person = Student.new(age, name, permission_values.include?(permission))
+    @all_persons.push(person)
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    person = Teacher.new(age, specialization, name)
+    @all_persons.push(person)
+  end
+
+  def list_books
+    puts @all_books.inspect
+    @all_books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
```

Comment:
- [OPTONAL] There is some line above the list of books that you need to remove when listing the books option one (1). 👇🏻 


![image](https://github.com/lucy-sees/Libra/assets/79058364/a4ff95b1-aaad-4e47-b780-7f6b0fca922c)